.github/actions/build-push-image/action.yaml

```diff
@@ -49,6 +52,7 @@ runs:
       uses: actions/checkout@v3.5.3
       if: ${{ inputs.forked == false }}
       with:
+        ref: ${{ inputs.ref }}
```

Comment:
If `ref` is not specified, which one is used? I'm asking because this action also used by the E2E tests job